src/assets/css/app.css

```diff
@@ -2873,7 +2873,7 @@ textarea.form-control-lg {
   width: 1rem;
   height: 1rem;
   margin-top: -0.25rem;
-  background-color: #31bb6b;
+  background-color: #1778f2;
```

Comment:
1. Should this be moved to `style/app.module.css` for the components? 
2. Is the Left Nav slider?